math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1628,7 +1628,7 @@ T InvariantMass(const RVec<T>& pt, const RVec<T>& eta, const RVec<T>& phi, const
 /// RVec<float> phis {.1f, 3.02f, 2.2f};
```

Comment:
```suggestion
/// RVec<float> phis = {.1f, 3.02f, 2.2f};
```

And the same for the `etas` above. I would also order them such as we use them in the `Construct` (so pt eta phi and mass).

Reply:
Also just probably my opinion: I would write out number like `0.1` rather than `1.f`. It's more readable (and we don't care here about the implicit conversion).